src/main/java/hw_3/Task7.java

```diff
@@ -0,0 +1,18 @@
+package hw_3;
+
+public class Task7 {
+        public static void main(String[] args) {
+            int[][] numbers = {
+                    {1, 2, 3},
+                    {4, 5, 6},
+                    {7, 8, 9}
+            };
+            for (int i = 0; i < numbers.length; i++) {
+                for (int j = 0; j < numbers[i].length; j++) {
+
+                    System.out.print(numbers[i][j] + " ");
+                }
+                System.out.println();
```

Comment:
THis System.out.println can be removed? If you need to type something from the new line you can use '\n'

Reply:
Done